std/encoding.d

```diff
@@ -652,7 +662,8 @@ template EncoderInstance(E)
 
 private template GenericEncoder()
 {
-    bool canEncode(dchar c)
+    @safe:
```

Comment:
Shouldn't the compiler be already inferring `@safe` here, since this is a template?  Looks like it may just be that one function that needed the `@trusted`.

A `@safe` unittest should be enough to ensure the code doesn't accidentally become un-`@safe`.